config/game_options.txt

```diff
@@ -1,4 +1,4 @@
-## HEALTH ###
+f## HEALTH ###
```

Comment:
Was the "f" in front of ## HEALTH ## intentional?

Reply:
you did this, not me!!!